src/functions/template-tags/date.php

```diff
@@ -290,6 +290,13 @@ function tribe_get_start_date( $event = null, $display_time = true, $date_format
 			return '';
 		}
 
+		static $start_dates = [];
```

Comment:
:no_entry_sign: Short array syntax is not allowed

<sub>hash: fbbfc6857d2901c877c3d425ef6887ac</sub>